change_log/next/dropdown_filter_ajax_request.yml

```diff
@@ -0,0 +1 @@
+Improvement: "Improved DropdownFilterAjax component Request handling"
```

Comment:
Can you add a bit more detail here so people know what to expect with the upgrade.